config/websockets.php

```diff
@@ -91,4 +91,9 @@
          */
         'passphrase' => null,
     ],
+
+    /*
+     * Channel Manager
```

Comment:
Add some more info here on what this does and in which cases you would change this option

Reply:
Updated. 

```
* This class handles how channel persistence is handled.
* By default, persistence is stored in an array by the running webserver.
* The only requirement is that the class should implement
* `ChannelManager` interface provided by this package.
```